src/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/config.go

```diff
@@ -24,19 +23,13 @@ type initContainerModifier interface {
 	getInitContainers() []corev1.Container
 }
 
-var (
-	log = logger.Factory.GetLogger("activegate-statefulset-builder")
```

Comment:
was this not used?

Reply:
Only by `ExtensionControllerModifier`, which got thrown out.